tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -166,6 +169,18 @@ Verify RHODS Release Version Number
     ${version} =  Get RHODS Version
     Should Match Regexp    ${version}    ^[0-9]+\.[0-9]+\.[0-9]+\(-[0-9]+)*$
 
+Verify Users Can Update Notification Email After Installing RHODS With The AddOn Flow
+    [Documentation]    Vrifies the Alert Notification email is updated in Addon-Managed-Odh-Parameters Secret and Alertmanager ConfigMap
+    [Tags]    Tier2
+    ...       ODS-673
+    ...       KnownIssues
+    ...       Deployment-AddOnFlow
+    ${email_to_change} =    Set Variable    dummyemail2@redhat.com
+    ${cluster_id} =    Get Cluster ID
+    ${cluster_name} =    Get Cluster Name By Cluster ID    ${cluster_id}
+    Update Notification Email Address    ${cluster_name}    ${email_to_change}
+    Wait Until Notification Email From Addon-Managed-Odh-Parameters Contains  email=${email_to_change}
+    Notification Email In Alertmanager ConfigMap Should Be    ${email_to_change}
```

Comment:
You will probably need a keyword `Wait Until Notification Email In Alertmanager ConfigMap Is` too

Reply:
Done
In Teardown I have added to restore email